README.md

```diff
@@ -158,7 +158,7 @@ Tag\|TagB                    |have either TagA or TagB.
 
```

Comment:
Let's change CHANNEL on line 156 to `stable`. The same applies to file `env/test/test.properties`